packages/single-line/src/SingleLineEditor.tsx

```diff
@@ -16,17 +16,22 @@ export interface SingleLineEditorProps {
   /**
    * is the field disabled initially
    */
-  isInitiallyDisabled: boolean;
+  isInitiallyDisabled?: boolean;
 
   /**
    * is the field manually disabled
    */
   isDisabled?: boolean;
 
+  /**
+   * whether validations should be rendered or not.
+   */
+  validationsAreVisable?: boolean;
```

Comment:
should we name it `withValidations`? to stay consistent in the naming

or `withCharInformation`? as the char counter would be no validation 🙈

Reply:
+1 to renaming to `withCharInformation` to match naming conventions of this component and it matches the purpose of the boolean a bit better since to Chris' point it's not just for validations but also the character counter.